cmd/client_test.go

```diff
@@ -111,3 +111,55 @@ func TestClient_CreateJobSpec(t *testing.T) {
 		assert.Equal(t, test.nJobs, len(numberOfJobs))
 	}
 }
+
+func TestClient_CreateJobRun(t *testing.T) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+	client, _ := cltest.NewClientAndRenderer(app.Store.Config)
+
+	tests := []struct {
+		json    string
+		jobType int
+		badID   bool
+		errored bool
+	}{
+		{`{"value": 100}`, 0, false, false},
+		{``, 0, false, false},
+		{`{`, 0, false, true},
+		{``, 0, true, true},
+		{``, 1, false, true},
+	}
+
+	for _, tt := range tests {
```

Comment:
Let's use golang's subtests as shown here: https://github.com/smartcontractkit/chainlink/blob/8e9fc48ed35899787f6891b3331dd89795442ee8/services/scheduler_test.go?utf8=%E2%9C%93#L57-L61

```golang
	for _, tt := range tests {
		test := tt // to capture variable before closure
		t.Run(test.name, func(t *testing.T) { // note this t.Run as a subtest
			r := services.NewRecurring(store)
			cron := cltest.NewMockCron()
```

More on subtests can be read here https://blog.golang.org/subtests
Be sure to scroll down about 3 sections.